core/unit_test/TestMDRangePolicyConstructors.hpp

```diff
@@ -93,6 +93,27 @@ TEST(TEST_CATEGORY_DEATH, policy_bounds_unsafe_narrowing_conversions) {
       },
       "unsafe narrowing conversion");
 }
+
+TEST(TEST_CATEGORY_DEATH, policy_invalid_bounds) {
+  using Policy = Kokkos::MDRangePolicy<TEST_EXECSPACE, Kokkos::Rank<5>>;
```

Comment:
Why rank 5 if we can trigger with 2?

Reply:
No real reason. Changed to 2 ranks and removed the second test case.